test-app-sitebricks.yaml

```diff
@@ -51,9 +51,9 @@ paths:
           description: "invalid id of user"
```

Comment:
I still can't see it, how are you expecting the id to be assigned to the user, where it comes from?